packages/amplify-graphql-api-construct-tests/src/__tests__/relationships/postgres-uuid-pk/uuid-pk-ddbprimary-sqlrelated.test.ts

```diff
@@ -23,8 +23,9 @@ describe('PostgreSQL tables with UUID primary keys', () => {
   const region = process.env.CLI_REGION ?? 'us-west-2';
   const baseProjFolderName = path.basename(__filename, '.test.ts');
 
+  // sufficient password length that meets the requirements for RDS cluster/instance
```

Comment:
nit: here and other places - don't think this comment is relevant here, since we're generating DB Name.